provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -1780,12 +1787,14 @@ rules:
   - watch
   - get
   - update
+{% if config.nodepodif_config.enable %}
```

Comment:
My suggestion is that we dont change any of this, we just take the flag from the user and relay it to the host-agent config. That way, the CRD and the necessary RBAC is already in place if this flag is flipped let's say directly in the config-map.

Reply:
sure, that makes sense, will update the PR shortly. Thanks for the review.